app/src/main/java/com/example/quizacademytask/StackListFragment.kt

```diff
@@ -220,6 +270,10 @@ class StackListFragment : Fragment(), SimpleAdapter.OnItemClickListener {
     }
 
     override fun onItemClick(position: Int, v: View?) {
+        if (actionMode && v != null) {
```

Comment:
better would be to remember selected position and use it in onBindViewHolder to highlight selected items. This way it would be easier to support restoring state after orientation changes